File: BackEnd/Models/Aviao.cs
using System.ComponentModel.DataAnnotations;

public class Aviao
{
    [Key]
    public int Id { get; set; }
    public string? Modelo { get; set; }
    public CompanhiaAerea? CompanhiaAerea { get; set; }
    public int CompanhiaAereaId { get; set; }
}

File: FrontEnd/AppPassagem/src/app/models/Aviao.ts
import { CompanhiaAerea } from "./CompanhiaAerea";

export class Aviao {
    id: number = 0;
    modelo: string = "";
    companhiaAerea : CompanhiaAerea;
    companhiaAereaId : number = 0;
    
    constructor(companhiaAerea : CompanhiaAerea){
        this.companhiaAerea = companhiaAerea;
    }
}
